### PR TITLE
google calendar integration + style things

### PR DESCRIPTION
**Pull request by cwRichardKim**
changed titles
google calendar integration + style things

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 
-<h2>Contact info</h2>
-* acmexecs@elist.tufts.edu
-* team@tufts.io
+<h2>Groups and Elists</h2>
+<ul>
+  <li>CS FB Groups:  <a href="https://www.facebook.com/TuftsCSX/?fref=ts">CSX</a>, <a href="https://www.facebook.com/groups/TuftsCS/">Tufts CS</a></li>
+  <li>ELS FB Groups: <a href="https://www.facebook.com/groups/217962551630823/">Entrep At Tufts</a>, Tufts Entrepreneurial Network (<a href="https://www.facebook.com/tuftsentrepreneurialprogram/?fref=ts">open</a>, <a href="https://www.facebook.com/groups/295587574102/">closed</a>)</li>
+  <li>Design FB Groups: <a href="https://www.facebook.com/groups/688983981203665/">Tufts Design Thinking</a></li>
+  <li>ELists: <a href="mailto:acmexecs@elist.tufts.edu?subject=Can I be added to the CSX Elist? (EOM)">CSX</a></li>
```
Comment (bsteephenson) [inline index.html]: Could we replace this with a link to the mailchimp signup page?

Comment (cwRichardKim) [reply index.html]: yea can someone who is in charge of that send me the link?

Comment (bsteephenson) [reply index.html]: http://alexlenail.us7.list-manage.com/subscribe?u=c6bc0c27749fcf9073bb37914&id=0ffb1e8a92

Comment (cwRichardKim) [reply index.html]: done. thanks for the suggestion

Comment (tekknolagi) [reply index.html]: Can we transfer that mailchimp account away from alexlenail.something?

**Pull request merged**